app/client/src/components/pages/MonitoringReport.js

```diff
@@ -1985,13 +1985,14 @@ function InformationSection({ siteId, site, siteStatus }) {
     <div css={modifiedBoxStyles}>
       <h2 css={infoBoxHeadingStyles}>
         <span>
-          {siteStatus === 'fetching' && <LoadingSpinner />}
-          {siteStatus === 'success' && site.locationName}
+          <small>
+            <strong>Location Name: </strong>
+            {siteStatus === 'fetching' && <LoadingSpinner />}
+            {siteStatus === 'success' && site.locationName}
+          </small>
```

Comment:
This works for me, but it doesn't align with the Waterbody Report page.

Reply:
Yeah it is a little different. We spoke with Kiki and she was ok with it not matching up with the waterbody report in this case. The main reason is because monitoring locations can have weird names that are just numbers, like 01651770. Waterbodies pretty much always have an obvious name.